content/projects/basic-unit-test-mocks-kata/_index.md

```diff
@@ -111,21 +111,43 @@ If you want some further reading check this out: {{% contentlink path="topics/li
 
 ## The actual project
 
-Write a program that sends a random inspirational quote to an email address, you should export 2 functions from a file named `sendEmail.js`:
-  1. `handleSendEmail` it should take in a parameter which is the recipients email address, and should be the only function responsible for sending the email via SMTP
-  2. `sendEmail` it should handle getting the recipients email address, the email address could come from the command line or passed directly to onSendEmail as a parameter
+Write a program that sends a random inspirational quote to an email address, you should export 2 functions from a file named `send_email.js`:
+
+1. `handleSendEmail` should take in a parameter which is the recipients email addresses, and should be the only function responsible for sending the email via SMTP.
```

Comment:
It might be good to say 

1.  The `handle send mail` function (use the appropriate naming conventions for the language you are using) should ...
2. etc